src/Symfony/Component/Form/Resources/translations/validators.fa.xlf

```diff
@@ -4,15 +4,135 @@
         <body>
             <trans-unit id="28">
                 <source>This form should not contain extra fields.</source>
-                <target>این فرم نباید شامل فیلد اضافه ای باشد.</target>
+                <target>این فرم نباید شامل فیلد اضافه‌ای باشد.</target>
```

Comment:
Is this correct? 

It should be plural. Ie more that one extra field.

Reply:
it's correct. in cases such as this one, persian is different than english.

Reply:
i can say it in another way that needs to be plural. but this structure dosn't need plural nouns. i will change it.